### DISCUSSION TITLE
Assimp exporter throws segmentation fault with .stp files

### DISCUSSION THREAD

**Discussion by IEI245MU5** [assimp/assimp#5094] [Q&A]
Hello,

I wrote a function that is supposed to convert a .stp file to .dae with the assimp importer/exporter:

```c++
void convertToDae(std::string pathToStp)
{
    Assimp::Importer importer;
    const aiScene* scene = importer.ReadFile(pathToStp, 
        aiProcess_CalcTangentSpace       | 
        aiProcess_Triangulate            |
        aiProcess_JoinIdenticalVertices  |
        aiProcess_SortByPType);

    if (!scene)
    {
        std::cout << "ERROR: model could not be loaded. \n";
        return;
    }

    std::cout << "Model was successfully loaded. \n";

    // export the scene as a Collada file
    Assimp::Exporter exporter;
    exporter.Export(scene, "collada", "../solutions/newPart.dae");

    return;
}
```

When I call this function inputting a .stp file I get the following output:
```
Model was successfully loaded. 
Segmentation fault (core dumped)
```
However when I input another file format like .dae it works as intended. Does this mean my .stp files are bad or do I have to adjust my program some way to accomodate .stp files?

Any help is greatly appreciated. Thank you :)

**Comment by kimkulling** [comment]
Stepfiles are currently not supported. The only format which is supported are IFC files for architectural data. I am wondering why the system does not tell this?
